### ISSUE TITLE
[navigator] single-clicking image/pdf files in navigator would open the source instead of the preview

### ISSUE THREAD

**Issue by fangnx** [eclipse-theia/theia#5230]
When we had `enablePreview: true` in the *preferences settings*, single-clicking files of supported image/pdf extentions (jpg, png, ...) in the navigator would open the source instead of the preview. Seems like the `EditorPreviewManager` takes precedence over the image/pdf handlers.

Opening a pdf & a jpg file by single-clicking:
![](https://media.giphy.com/media/f7jGxLdB34Oo5vz41P/giphy.gif)

**Comment by kittaakos** [comment]
It works for me. I am not on the `master` though.

![screencast 2019-05-23 17-43-07](https://user-images.githubusercontent.com/1405703/58266671-7265b780-7d82-11e9-85a7-e4cf8b052c37.gif)

Opening a `.jpg` has some issues:
```
mini-browser-content.ts:679 Resource interpreted as Document but transferred with MIME type image/jpeg: "http://localhost:3001/mini-browser/Users/akos.kitta/Desktop/vscode/src/vs/base/test/node/encoding/fixtures/some.json.png"
```

**Comment by kittaakos** [comment]
Since you have updated the title, I have checked with a single click too, it workd.

**Comment by fangnx** [comment]
> Since you have updated the title, I have checked with a single click too, it worked.
> 
> Edit: typo.

I just realized it is related to the *preferences setttings*. I got this issue when I had `enablePreview: true `.

**Comment by kpge** [comment]
I think this bug is cased by editor preview , you can see my pr to fix it: #5238

**Comment by kittaakos** [comment]
> I just realized it is related to the _preferences setttings_. I got this issue when I had `enablePreview: true `.

![Screen Shot 2019-05-27 at 11 42 48](https://user-images.githubusercontent.com/1405703/58411447-98d26e00-8074-11e9-84b0-7f38f8c8dd1c.jpg)

What about this preference? I thought this should control how we open previews, the `enablePreview` is used for whether we want to reuse a single "preview" code editor or not.

I have checked #5243. I set `"preview.openByDefault": false` and when I open a PNG, it opens the preview instead of the source.
